sound/soc/intel/boards/sof_es8336.c

```diff
@@ -143,6 +120,7 @@ static int sof_8336_trigger(struct snd_pcm_substream *substream, int cmd)
 			if (substream->stream == 0) {
 				cancel_delayed_work(&priv->pcm_pop_work);
 				gpiod_set_value_cansleep(priv->gpio_speakers, true);
+				gpiod_set_value_cansleep(priv->gpio_headphone, false);
 			}
```

Comment:
Turning off the headphone GPIO reduces pop/click noise and hum noise. For suspend I mean deactivating a steam. And from here we can see the headphone GPIO is on even the stream deactivated.